oe_paragraphs.post_update.php

```diff
@@ -23,3 +25,13 @@ function oe_paragraphs_post_update_contextual_navigation_fix_description(array &
   $field->set('description', 'The number of items to display. When empty, defaults to 4.');
   $field->save();
 }
+
+/**
+ * Make oe_links required in contextual navigation.
+ */
+function oe_paragraphs_post_update_00001_field_oe_links(array &$sandbox): void {
+  $field = FieldConfig::load('paragraph.oe_contextual_navigation.field_oe_links');
+  $field->setRequired(TRUE);
+  $field->setSetting('title', 1);
```

Comment:
The value of this property needs to be 2 so the title is required.

Reply:
Fixed.